lib/models/store_product_wrapper.dart

```diff
@@ -38,6 +38,9 @@ class StoreProduct with _$StoreProduct {
     @JsonKey(name: 'discounts', nullable: true)
         List<StoreProductDiscount>? discounts,
 
+    /// Product type. Null for iOS.
+    @JsonKey(name: 'productCategory', nullable: true) PurchaseType? productType,
```

Comment:
is now a good time to unify the naming of these? or do you think it should stay..

Reply:
I'm so torn! I think we could do the "productCategory" name change at a different time (because we do lock into specific PHC versions) so that could be a future change.

I don't _love_ `PurchaseType` for this and think it probably should be `ProductType` and... now is probably a good time to deprecate it and make a `ProductType` 🤷‍♂️  I'll make that change!